var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -76,6 +77,12 @@ class Proj(CMakePackage, AutotoolsPackage):
         when="@:6",
     )
 
+    patch(
+        "https://github.com/OSGeo/PROJ/commit/3f38a67a354a3a1e5cca97793b9a43860c380d95.patch?full_index=1",
+        sha256="ff09e573b54c41208899b4abd66a494d3e0a3fbafdb0d024a2a262c22b9526b6",
```

Comment:
```suggestion
        sha256="dc620ff1bbcc0ef4130d53a40a8693a1e2e72ebf83bd6289f1139d0f1aad2a40",
```
Adding the suffix changes the file